### PR TITLE
Add remaining sidebar metric components

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard8**
## Summary

This PR adds the remaining sidebar metric components. For behavior, I had to refactor a bit in order to allow the sidebar to show data for a metric without referring to a specific property. Not sure if it was worth it for just the sidebar, but for choropleths we probably also need this type of exception if we start applying the metric-config there too.

### CODE REVIEW

**Review by VWSCoronaDashboard1 — APPROVED** [review]

**Review by VWSCoronaDashboard10 — APPROVED** [review]

File: src/components-styled/sidebar-metric/sidebar-kpi-value.tsx [hunk 1]
```diff
@@ -8,7 +8,11 @@ import { Heading, InlineText } from '../typography';
 
 type SidebarKpiValueProps = {
   title: string;
-  value: number;
+  /**
+   * Make value optional for odd case where we do not show a metric.
+   * Currently only behavior is doing that.
```
Comment (VWSCoronaDashboard4) [inline src/components-styled/sidebar-metric/sidebar-kpi-value.tsx:13]:
```suggestion
   * Currently only 'Behavior' is doing that.
```

File: src/components-styled/sidebar-metric/sidebar-kpi-value.tsx [hunk 2]
```diff
@@ -38,12 +42,25 @@ export function SidebarKpiValue(props: SidebarKpiValueProps) {
...
+            marginRight={isDefined(difference) ? 1 : 3}
+          >
+            {isPercentage ? `${formatPercentage(value)}%` : formatNumber(value)}
+          </InlineText>
+        )}
 
         {isDefined(difference) && (
-          <Box as="span" fontSize={3} display="flex" alignItems="center">
+          <Box
+            as="span"
```
Comment (VWSCoronaDashboard4) [inline src/components-styled/sidebar-metric/sidebar-kpi-value.tsx:58]:
```suggestion
```

`DifferenceIndicator` contains block elements, so a div is better (which is default for Box)

File: src/components-styled/sidebar-metric/sidebar-metric.tsx [hunk 3]
```diff
@@ -14,7 +14,11 @@ interface SidebarMetricProps<T extends { difference: unknown }> {
   scope: DataScope;
   data: T;
   metricName: ValueOf<MetricKeys<T>>;
-  metricProperty: string;
+  /**
+   * Make metric property optional for odd case where we do not show a metric.
+   * Currently only behavior is doing that.
```
Comment (VWSCoronaDashboard4) [inline src/components-styled/sidebar-metric/sidebar-metric.tsx:19]:
```suggestion
   * Currently only 'Behavior' is doing that.
```

I misread 'Behavior' as part of the comment, not as reference.

**Review by VWSCoronaDashboard4 — APPROVED** [review]
Looking good! A nitpick about a comment, and a suggestion to convert span to div.

File: src/components-styled/sidebar-metric/sidebar-kpi-value.tsx (hunk 1 above)
Comment (VWSCoronaDashboard8) [reply src/components-styled/sidebar-metric/sidebar-kpi-value.tsx:13]: Sorry I missed your comments. I'll make the changes on develop

**Pull request merged**